spec/fixtures/.haml-lint.yml

```diff
@@ -0,0 +1,7 @@
+linters:
+  RuboCop:
+    enabled: true
+    ignored_cops:
+      - Style/FinalNewline
+      - Style/TrailingBlankLines
+      - Style/EndOfLine
```

Comment:
For some reason this rule was triggering on all of the files for me, causing issues for the "valid" one. Unfortunately even though the other two are defined in the defaults and shouldn't be necessary here, it doesn't work without them.